### ISSUE TITLE
Visning af et resultat som fanges af flere searchere

### ISSUE THREAD

**Issue by furesoe** [Septima/OneDoor#45]
@klavspc @rastermanden

Kunne det laves sådan at det var mere tydeligt, hvilken searcher man vil se resultat fra, når et søgeord fanges af flere searchere.

I nedenstående billede finder søgeordet et resultatbåde i lokalplaner, stednavne og kommunale ejendomme (en searcher baseret på eget data-lag).

Jeg kan se, at der ud for resultatet fra lokalplan-searcheren angives at det er en 'Vedtaget lokalplan'.

Kunne man lave noget tilsvarende på alle andre searchere? At der fx stod 'Stednavne' eller 'Kommunale ejendomme' i parantes efter dem?

![image](https://user-images.githubusercontent.com/38730459/219036466-40799ef9-8bfa-4356-a9b2-3c8cdc3cf426.png)

I det hele taget er der nogle overvejelser at gøre sig med at få mange døre ind til (i bund og grund) den samme lokalitet.

Hos mig vil den samme søgning kunne finde en forekomst i:

- Stednavne
- Kommunale ejendomme
- Kommunale P-enheder
- Produktionsenheder
- Vedtagne lokalplaner

**Comment by klavspc** [comment]
Ja, det kunne vi godt arbejde lidt mere med. Det bliver dog ikke i allenærmeste fremtid pga mange opgaver med højere prioritet

Kan du uddybe andel del efter "i det hele taget...."?

**Comment by furesoe** [comment]
@klavspc 

Jeg tænkte blot, at det at man kan finde den samme lokalitet via 5 indgange (stednavn, P-nummer og hvad man ellers bygger op af data selv (Kommunale ejendomme/Kommunale P-enheder)), strider lidt imod tanken om "one door".

Jeg tænker bare om tanken kunne dyrkes omkring om man kunne udnytte én af searcherne (fx stednavne) til at komme videre derfra. Men det vil  nok stadig betyde at man misser nogle vigtige lokaliteter.

Jeg ved ikke om jeg kan komme det nærmere lige nu...

**Comment by DocDemar** [comment]
> Jeg tænker bare om tanken kunne dyrkes omkring om man kunne udnytte én af searcherne (fx stednavne) til at komme videre derfra. Men det vil nok stadig betyde at man misser nogle vigtige lokaliteter.

Dette kan konfigureres i OneDoor med SQ handler som eksempelvis kunne søge i lokalplaner og P-enheder

**Comment by DocDemar** [comment]
> Jeg tænkte blot, at det at man kan finde den samme lokalitet via 5 indgange (stednavn, P-nummer og hvad man ellers bygger op af data selv (Kommunale ejendomme/Kommunale P-enheder)), strider lidt imod tanken om "one door".

Det er et problem når forskellige typer har samme navn.
Vi har mulighed for at skrive i resultatet hvilket register resultatet stammer fra
Hvad tænker du om den løsning?
